test/server_test.py

```diff
@@ -444,14 +448,13 @@ def emailAdded():
 			assert_that(user.email, equal_to(test_email))
 			assert_that(Code.get_by_code(test_code), none())
 
-	@describe('Remove email from user', only=True)
```

Comment:
I really should have asked what this was before I approved it last time. `only=True`